clusterloader2/pkg/prometheus/manifests/prometheus-clusterRole.yaml

```diff
@@ -1,19 +1,19 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  labels:
+    app.kubernetes.io/component: prometheus
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 2.24.0
   name: prometheus-k8s
 rules:
 - apiGroups:
   - ""
   resources:
   - nodes/metrics
-  - services
-  - endpoints
-  - pods
   verbs:
   - get
-  - list
-  - watch
```

Comment:
Are you sure we can remove this? 
If I am not mistaken when we are using service/pod monitors prometheus is doing list and watch for these two objects, right?